test/util/netLaunch/index.ts

```diff
@@ -153,6 +153,14 @@ export class NetworkLauncher {
       composeYaml.services["sh-user"].command.push(
         "--database-url=postgresql://postgres:postgres@docker-sh-postgres-1:5432/storage_hub"
       );
+      if (this.type === "fullnet") {
+        composeYaml.services["sh-msp-1"].command.push(
+          "--database-url=postgresql://postgres:postgres@docker-sh-postgres-1:5432/storage_hub"
+        );
+        composeYaml.services["sh-msp-2"].command.push(
+          "--database-url=postgresql://postgres:postgres@docker-sh-postgres-1:5432/storage_hub"
+        );
+      }
```

Comment:
We may want to add this just in the yml files directly. I don't think we need to do it dynamically, as MSPs will always need that indexer connection.

Reply:
This would break for tests where we don't need the indexer but use MSPs - we can not support that case and always have an indexer instead.